polkadot-wiki/sidebars.js

```diff
@@ -42,6 +41,7 @@ module.exports = {
           items: [
             "general/staking-apps",
             "general/governance-apps",
+            "general/multisig-apps",
```

Comment:
I thought the consensus in our TechEd chat was that multisigs are grouped under wallets, right?